Tutorials/Creating-a-Custom-Dashboard/index.md

```diff
@@ -165,7 +181,7 @@ We register this AngularJS controller to the Umbraco Angular module:
 ```js
 angular.module("umbraco").controller("CustomWelcomeDashboardController", function ($scope) {
     var vm = this;
-    alert('hello world');
+    alert("hello world");
```

Comment:
In line 195:
Sentence is a bit weird.
Should it be something like this instead perhaps:
"The use of vm (short for view model) is to enable ..."